### DISCUSSION TITLE
Docker ?

### DISCUSSION THREAD

**Discussion by piraterx** [boojack/sticky-notes#3] [General]
I can't believe what  have been looking for exist but not expert on how to host. I know docker. Do I host this as wordpress site behind webserver? 

I have been using mojopaste for quick ideas till I learn about memos today morning. First thing I did was to up and run then I learn about sticky notes. Really good project. 

Any way to make this docker file?

**Comment by jxpsert** [comment]
You can’t host memos on a webserver like HTML and PHP files. You’ll need a server with command-line access.

**Reply by piraterx** [reply]
> You can’t host memos on a webserver like HTML and PHP files. You’ll need a server with command-line access.

Memos, I was able to do with docker. That's fine. I am talking about sticky notes. If there is docker option?

**Reply by soundneedle** [reply]
wondering same

**Comment by jxpsert** [comment]
Oh, my bad. Didn’t know I was watching sticky notes and thought this was the memos repo.
________________________________
Van: piraterx ***@***.***>
Verzonden: Monday, June 12, 2023 1:49:51 AM
Aan: boojack/sticky-notes ***@***.***>
CC: Jasper Platenburg ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [boojack/sticky-notes] Docker ? (Discussion #3)


You can’t host memos on a webserver like HTML and PHP files. You’ll need a server with command-line access.

Memos, I was able to do with docker. That's fine. I am talking about sticky notes. If there is docker option?

**Comment by piraterx** [comment]
@boojack how do I self-host it? I feel like this is best solution for what I am looking for?
